doc/examples/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst

```diff
@@ -129,7 +127,7 @@ Planning results in ``moveit_msg::MotionPlanResponse``
 The LIN motion command
 ----------------------
 
-This planner generates linear Cartesian trajectory between goal and
+This planner generates a linear Cartesian trajectory between goal and
 start poses. The planner uses the Cartesian limits to generate a
 trapezoidal velocity profile in Cartesian space. The translational
 motion is a linear interpolation between start and goal position vector.
```

Comment:
```suggestion
motion is a linear interpolation between the start and goal position vector.
```
Same applies for the line below about rotational motion, but GitHub doesn't allow me to leave a suggestion there.